src/main/java/redis/clients/jedis/Protocol.java

```diff
@@ -253,7 +253,7 @@ private Command() {
 
     AGGREGATE, ALPHA, ASC, BY, DESC, GET, LIMIT, NO, NOSORT, ONE, SET, STORE, WEIGHTS, WITHSCORES,
     RESETSTAT, REWRITE, RESET, FLUSH, EXISTS, LOAD, KILL, LEN, REFCOUNT, ENCODING, IDLETIME, DOCTOR,
-    HELP, FREQ, GETNAME, SETNAME, LIST, MATCH, COUNT, TYPE, UNLOAD, REPLACE, PAUSE, BLOCK, NOACK,
+    HELP, FREQ, GETNAME, SETNAME, LIST, MATCH, COUNT, ANY, TYPE, UNLOAD, REPLACE, PAUSE, BLOCK, NOACK,
```

Comment:
Put ANY at the list line.

Reply:
What is the list line? Do you mean at the end?

Reply:
Oops, sorry. "Last" line.

Reply:
You've changes it already 😃